tools/Gemfile.plugins.test

```diff
@@ -14,3 +18,10 @@ plugins = [ 'logstash-filter-clone',
 plugins.each do |plugin|
   gem plugin
 end
+
+##
+# Dependencies related with coverage analysis
+##
+
+gem 'simplecov'
+gem 'coveralls'
```

Comment:
Not sure if this is the right place to declare the dependencies?
I would think the `logstash.gemspec` would be a better place for them?